hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -80,13 +80,18 @@ public SCMStateMachine(final StorageContainerManager scm,
     }
   }
 
+  public SCMStateMachine() {
```

Comment:
Instead of doing this, can we create StateMachinee Object like normal creation flow, so that we don' need to add any null checks, as all objects will be null.

I see from code to do this, we need to create metadata store and SCM, do you think we need to do this way, to handle any NPE, if any other method is called.

Reply:
I think its ok to not create instantiate any SCM sever instance. If any other method gets called suring ratis server spin up, we need to handle that in SCM anyways..

Reply:
Fine with me, as now no usage of ratis server instances we should be fine for now.